public/app/features/provisioning/utils/routes.ts

```diff
@@ -3,10 +3,11 @@ import { SafeDynamicImport } from 'app/core/components/DynamicImports/SafeDynami
 import { RouteDescriptor } from 'app/core/navigation/types';
 import { DashboardRoutes } from 'app/types';
 
+import { requiredFeatureToggles } from '../Setup/types';
 import { PROVISIONING_URL } from '../constants';
 
 export function getProvisioningRoutes(): RouteDescriptor[] {
-  if (!config.featureToggles.provisioning) {
+  if (requiredFeatureToggles.every((toggle) => config.featureToggles[toggle])) {
```

Comment:
I think this condition is the other way around. It should be kicking if any is missing. Btw, there is a function you can use to check for this in `types` so that you don't have to operate on the list directly.